huggle/loadingform.cpp

```diff
@@ -43,6 +43,11 @@ void LoadingForm::Info(QString text)
     this->ui->label_2->setText(text);
 }
 
+void LoadingForm::AbortLabel(QString text)
```

Comment:
I just recopied the `Info` above.
How can we change the label of an object from an other file/class without using an intermediary function ?